tests/Api/executeToggleTaskDoneCommand.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import moment from 'moment';
+import type { App } from 'obsidian';
+import { tasksApiV1 } from '../../src/Api';
+
+jest.mock('obsidian', () => ({
+    Modal: class Mock {},
+}));
+
+window.moment = moment;
+
+const app = {} as App;
```

Comment:
Wow, this is nice. Did I understand correctly that the `app` parameter is not used at all here, so we can pass a `{} as App`?

Why do we need to mock the `Modal` from `obsidian`?

Just curious for science here =)

Reply:
Good question @ilandikov. I've added a comment to clarify. Through a chain of imports, the `Api.ts` file imports `TaskModal` which trys to extend Obsidian's `Modal` class when the test runs.

Also I've removed the `app` variable and inlined `{} as App` as you suggested

Reply:
> Also I've removed the `app` variable and inlined `{} as App` as you suggested

Thanks. My point about this was actually more generic. The point is, we don't use any `App` object in the call of `executeToggleTaskDoneCommand()` so for this reason we don't mock the `App` but just pass `{} as App` to the `tasksApiV1()`?